HW_7/task1.py

```diff
@@ -0,0 +1,22 @@
+# Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм. Поскольку разобраться в его кричалках не настолько просто, 
+# насколько легко он их придумывает, Вам стоит написать программу. Винни-Пух считает, что ритм есть, если число слогов 
+# (т.е. число гласных букв) в каждой фразе стихотворения одинаковое. Фраза может состоять из одного слова, если во фразе несколько слов, 
+# то они разделяются дефисами. Фразы отделяются друг от друга пробелами. Стихотворение  Винни-Пух вбивает в программу с клавиатуры. 
+# В ответе напишите “Парам пам-пам”, если с ритмом все в порядке и “Пам парам”, если с ритмом все не в порядке
+
+def rhyme(song):
+    str = song.lower().split()
+    f = lambda x: sum(1 for i in x if i in "аеёиоуыэюя")
+    tmp = f(str[0])
+    if all([f(i) == tmp for i in str]):
```

Comment:
Можно просто передать генератор
```suggestion
    if all(f(i) == tmp for i in str):
```